atomictests.py

```diff
@@ -6,6 +6,16 @@
 
 
 @unittest.skipUnless(if_atomic(), "It's not an atomic image")
+class TestAtomic01Status(unittest.TestCase):
+
+    def test_print_status(self):
+        """Print the rpm-ostree status"""
+        # We want this explicitly output to stdout
+        out, err, eid = system('rpm-ostree status')
+        print(out)
```

Comment:
@puiterwijk This is going to print `byte` string. You may want to convert it into `unicode` string?
Something similar like `out = out.decode('utf-8')` before printing the output.